### DISCUSSION TITLE
The command "backup:run" does not exist.

### DISCUSSION THREAD

**Discussion by tamim-ikbal** [spatie/laravel-backup#1265] [Q&A]
Hi spatie, I have faced a problem when I am run the command from my code, My code is fine, I have test the code, Because when I run this `Artisan::call('view:clear');`  then its working, But when I run this `Artisan::call('backup:run'); ` Browser told me **The command "backup:run" does not exist.**
**Note:** when I am run this `php artisan backup:run` using cmd then it's working, I will be grateful, If you guys help me, Thank you
Here is my All code of controller: 

`<?php

namespace App\Http\Controllers;

use Illuminate\Http\Request;
use Illuminate\Support\Facades\Artisan;

class BackupController extends Controller
{
    /**
     * Handle the incoming request.
     *
     * @param \Illuminate\Http\Request $request
     * @return \Illuminate\Http\Response
     */
    public function __invoke(Request $request)
    {
        try {
            Artisan::call("backup:run");

            return "Done";

        } catch (\Exception $e) {

            return $e->getMessage();
        }
    }
}
`

**Comment by HDVinnie** [comment]
Also getting this issue when calling `Artisan::call('backup:run')`

Started when upgraded from `7.2.1` to `7.3.0`

**Reply by HDVinnie** [reply]
Well downgrading didn't help and only other update was Laravel `8.29.0` to `8.32.1`. So @freekmurze its possible Laravel itself broke your package.

**Reply by wadchathuranga** [reply]
I am also getting this error. Is there anyone who knows to fix this?
